controllers/machinehealthcheck_controller.go

```diff
@@ -256,12 +258,58 @@ func (r *MachineHealthCheckReconciler) reconcile(ctx context.Context, logger log
 	for _, t := range unhealthy {
 		condition := conditions.Get(t.Machine, clusterv1.MachineHealthCheckSuccededCondition)
 
-		if annotations.IsPaused(cluster, t.Machine) {
-			logger.Info("Machine has failed health check, but machine is paused so skipping remediation", "target", t.string(), "reason", condition.Reason, "message", condition.Message)
+		if m.Spec.RemediationTemplate != nil {
+			// If external remediation request already exists,
+			// return early
+			if r.externalRemediationRequestExists(ctx, m, t) {
+				logger.V(2).Info("External Remediation Request already exists", "machine", t.Machine.Name, "templateGVK", m.Spec.RemediationTemplate.GroupVersionKind())
+				return ctrl.Result{}, nil
+			}
+
+			cloneOwnerRef := &metav1.OwnerReference{
+				APIVersion: clusterv1.GroupVersion.String(),
+				Kind:       t.Machine.Kind,
+				Name:       t.Machine.Name,
+				UID:        t.Machine.UID,
+			}
+
+			from, err := external.Get(ctx, r.Client, m.Spec.RemediationTemplate, t.Machine.Namespace)
+			if err != nil {
+				return ctrl.Result{}, errors.Wrapf(err, "Remediation template not found %q", m.Spec.RemediationTemplate)
+			}
+
+			generateTemplateInput := &external.GenerateTemplateInput{
+				Template:    from,
+				TemplateRef: m.Spec.RemediationTemplate,
+				Namespace:   t.Machine.Namespace,
+				ClusterName: t.Machine.ClusterName,
+				OwnerRef:    cloneOwnerRef,
+			}
+			to, err := external.GenerateTemplate(generateTemplateInput)
+			if err != nil {
+				return ctrl.Result{}, errors.Wrapf(err, "Failed to create template for remediation request %q", m.Spec.RemediationTemplate)
```

Comment:
```suggestion
				return ctrl.Result{}, errors.Wrapf(err, "error creating remediation request for template %q", m.Spec.RemediationTemplate)
```

plus same comment about splitting up that single `%q`

Should also probably include namespace, cluster name, machine name in the error message?

Do we surface this error in a condition anywhere? We need to make sure the log is not the sole place it goes.